symbolic-minidump/src/processor.rs

```diff
@@ -1031,6 +1034,34 @@ struct SymbolEntry {
 /// [`CodeModule`]: struct.CodeModule.html
 pub type FrameInfoMap<'a> = BTreeMap<CodeModuleId, CfiCache<'a>>;
 
+/// Options passed to breakpad when processing a minidump.
+#[derive(Clone, Debug)]
+#[non_exhaustive]
+pub struct ProcessMinidumpOptions {
+    /// The maximum number of threads the minidump may contain.
+    ///
+    /// Defaults to 4096.
```

Comment:
I think thats a lot higher already than the default in breakpad. Or do we intentionally increase all these?